internal/app/machined/pkg/controllers/time/time.go

```diff
@@ -0,0 +1,6 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// Package time contains controllers managing time, synchronization, etc.
+package time
```

Comment:
Do we need that file with _just_ a package description? Should it be moved to the main file?

Reply:
That's the style I tried to follow - have a file matching package name for docs and very global definitions for the package